examples/truffle/scripts/networks.js

```diff
@@ -0,0 +1,94 @@
+const fs = require("fs").promises;
```

Comment:
Why copy this file and not use the node dependency?

Reply:
I have to do some extra things besides just injecting networks. I also need to strip some non-reproducible bits that truffle inserts such as `updatedAt` and some more complicated metadata bits (like swarm hash of `metadata` property that gets embedded in the bytecode and is dependent on the absolute path of the contract when built).

Reply:
Oh! there is a utility library that does the network inject stuff for me! I'll use that instead.